CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# :arrow_down:Contributing Guidelines
+
+
+Follow these steps:hammer: for contributing:
+:green_heart:Fork the repo.
+
+:pushpin:Create a separate branch for any bug/feature with branch name: branch_<feature_name>
+
+:white_check_mark:Commit once a feature is updated/bug is fixed.
+
+:pencil2:Fetch for any changes in the original repo by creating an upstream using:
+
+>git remote add upstream https://github.com/codeforcauseorg/edu-server.git
+
+>git fetch upstream
+
+:tada:Merge the changes in your branch.
+
+>git merge upstream/(your_branch_name)
+
+:alien:Push your changes to your branch.
+
+:wrench:Make a pull request from your branch.
+
+:heavy_plus_sign:Before submitting a PR, check your code well and explain your work properly in the PR description.
+
+
+
+# :bookmark:Code advice
+
+Kindly maintain a clean code with proper comments.
+
+Commit messages should be written carefully and describe the changes made.
+
+Create separate PR for a single feature update/bug fix.
+
```

Comment:
Add some space after the emojis as they are overlapping with other text

![Screenshot 2021-03-16 at 4 57 45 PM](https://user-images.githubusercontent.com/51474217/111302199-1b2c8780-8679-11eb-9d45-d97e0adb7a5a.png)

Reply:
Sure!